portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,35 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+async function getData() {
+    const response = await fetch('/data');
+    const comments = await response.text();
+    JSON.parse(comments).forEach(comment => {
+        document.getElementById('data-container').appendChild(createListElement(comment.comment));
+    })
+}
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function createListElement(text) {
+    const liElement = document.createElement('li');
+    liElement.innerText = text;
+    liElement.classList.add('list-group-item', 'list-group-item-light');
+    return liElement;
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let i = 0;
+let text = "Learn to Code in HTML, CSS, JavaScript, jQuery, Node, Python, and Java";
```

Comment:
Generally speaking you want to default to `const` for variables unless you explicitly intend to reassign them later.